db/migrate/20190604144016_create_posts.rb

```diff
@@ -0,0 +1,9 @@
+class CreatePosts < ActiveRecord::Migration[5.2]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.string :link
```

Comment:
Should either of these fields be required? We usually like to have as much data integrity as possible enforced at the database level. If a post must have a title, you would do something like this here instead:
```
t.string :title, null: false
```